processor/src/app.ts

```diff
@@ -1,8 +1,8 @@
 import express from 'express'
 
-import { createProjectsAPI } from './projectAPI'
-import { createRemoteAPI } from './remoteAPI'
-import { createWorkers } from './workers'
+import { createProjectsAPI } from './projectAPI.js'
+import { createRemoteAPI } from './remoteAPI.js'
+import { createWorkers } from './workers.js'
```

Comment:
I ran into this as well and didn't like this solution. It seems so strange to add .js at the end and it breaks a bunch of stuff since that file doesn't exist a lot of the time. Is there really no better way?

Reply:
I don't like too but there are no [other options](https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#ecmascript-module-support-in-node-js). 

![image](https://user-images.githubusercontent.com/37152329/189124762-68ea865a-3c69-4eab-9f14-de921fd1de85.png)